main.go

```diff
@@ -70,7 +71,11 @@ func main() {
 	getopt.Parse()
 	args := getopt.Args()
 
-	homedir := lib.GetHomeDirectory()
```

Comment:
It's probably worth of 1) either switching to `lib.GetHomeDirectory()` for all `homedir.Dir()`+`log.Fatal()` occurrences (there are only a few use cases in current tests where `os.Exist(1)` is not used for failures getting user's home dir) 2) or removing `lib.GetHomeDirectory()` at all as not used anymore.
On one hand the former seems to make sense to unify use of the same code snippet across code base, though on the other hand some say it is best practice to avoid use of `log.Fatal` and, when used, should better be used directly from `main.go` only (as opposite to using it from non-main).

What do you think?

Reply:
I would prefer to delete the `lib.GetHomeDirectory()` function, since it does not add much over the base library. I'll push that update now. Hope with this update, we can get this merged. Thanks!